README.md

```diff
@@ -34,9 +34,8 @@ Its features for the electro-magnetic PIC algorithm include:
   [*Yee's*](http://dx.doi.org/10.1109/TAP.1966.1138693) and
   [*Lehe's*](http://dx.doi.org/10.1103/PhysRevSTAB.16.021301) scheme
 - rigorously charge conserving current deposition schemes, such as
-  [*Villasenor-Buneman*](http://dx.doi.org/10.1016/0010-4655%2892%2990169-Y),
   [*Esirkepov*](http://dx.doi.org/10.1016/S0010-4655%2800%2900228-9)
```

Comment:
Is this `EsirkepovNative` and should thus be removed?

Reply:
No, we still have the (optimized) Esirkepov - thus we should definitely keep the reference. 
`EsirkepovNative` is just an Esirkepov without optimization for developing purposes